tools/gcs-usage-recommender/python/backfill.py

```diff
@@ -63,18 +64,27 @@ def get_buckets(project_ids: List[str],
     output_list = []
     try:
         for project_id in project_ids:
-            bucket_list = list(gcs_client.list_buckets(project=project_id))
-            for bucket in bucket_list:
-                output_list.append({
-                    "bucket_name": bucket.name,
-                    "project_id": project_id,
-                    "last_read_timestamp": "",
-                    "days_since_last_read": -1,
-                    "read_count_30_days": -1,
-                    "read_count_90_days": -1,
-                    "export_day": datetime.datetime.utcnow().strftime("%Y-%m-%d"),
-                    "recommended_OLM": ""
-                })
+            try:
+                bucket_list = list(gcs_client.list_buckets(project=project_id))
+                for bucket in bucket_list:
+                    output_list.append({
+                        "bucket_name": bucket.name,
+                        "project_id": project_id,
+                        "last_read_timestamp": "",
+                        "days_since_last_read": -1,
+                        "read_count_30_days": -1,
+                        "read_count_90_days": -1,
+                        "export_day": datetime.datetime.utcnow().strftime("%Y-%m-%d"),
+                        "recommended_OLM": ""
+                    })
+            except Forbidden as err:
+                logging.error(f"You do not have access on the {bucket.name}.")
+                logging.error(err)
+                pass
```

Comment:
is this pass statement necessary?

Reply:
In the case of an org that has 1000s of buckets, the script would fail right now if there was one bucket that they didn't have access to. To keep the script running and just log the error, was using a pass to still work on other buckets.

Open to other ideas, lmk what you think @jaketf

Reply:
What I mean is you can omit this `pass` without changing behavior.
It doesn't do anything.

```
jferriero-macbookpro:~ jferriero$ cat scratch.py
try:
    raise ValueError
except Exception:
    print("caught exception with pass")
    pass
print("continued with pass")
try:
    raise ValueError
except Exception:
    print("caught exception without")
print("continued without pass")
jferriero-macbookpro:~ jferriero$ python3 scratch.py
caught exception with pass
continued with pass
caught exception without
continued without pass
```

Reply:
Got it - updated